app/src/main/java/com/amazonaws/kinesisvideo/demoapp/activity/WebRtcActivity.java

```diff
@@ -350,6 +344,7 @@ private void handlePendingIceCandidates(final String clientId) {
         while (pendingIceCandidatesQueueByClientId != null && !pendingIceCandidatesQueueByClientId.isEmpty()) {
             final IceCandidate iceCandidate = pendingIceCandidatesQueueByClientId.peek();
             final PeerConnection peer = peerConnectionFoundMap.get(clientId);
+            assert peer != null;
```

Comment:
I don't think assertions are a problem. The documentation you pointed out is for Java 8

Reply:
Do you have anything newer supporting that assert statements are not disabled by default? That's the latest official documentation I was able to find.

Reply:
It is a programming practice recommended by Android Studio since there were no null checks in our application. Also, I checked that it is enabled by default on Android. I explicitly set peer to null to see if this path gets invoked and it does:
```
2024-04-10 10:39:54.143 15976-16063 AndroidRuntime          com...nesisvideo.webrtc_sdk_android  E  FATAL EXCEPTION: OkHttp Dispatcher
                                                                                                    Process: com.amazonaws.kinesisvideo.webrtc_sdk_android, PID: 15976
 java.lang.AssertionError
at com.amazonaws.kinesisvideo.demoapp.activity.WebRtcActivity.handlePendingIceCandidates(WebRtcActivity.java:350)
at com.amazonaws.kinesisvideo.demoapp.activity.WebRtcActivity.access$700(WebRtcActivity.java:105)
at com.amazonaws.kinesisvideo.demoapp.activity.WebRtcActivity$1.onSdpAnswer(WebRtcActivity.java:251)
at com.amazonaws.kinesisvideo.signaling.SignalingListener$1.onMessage(SignalingListener.java:47)
at com.amazonaws.kinesisvideo.signaling.okhttp.WebSocketClient$1.onMessage(WebSocketClient.java:46)
at okhttp3.internal.ws.RealWebSocket.onReadMessage(RealWebSocket.kt:333)
at okhttp3.internal.ws.WebSocketReader.readMessageFrame(WebSocketReader.kt:245)
at okhttp3.internal.ws.WebSocketReader.processNextFrame(WebSocketReader.kt:106)
at okhttp3.internal.ws.RealWebSocket.loopReader(RealWebSocket.kt:293)
at okhttp3.internal.ws.RealWebSocket$connect$1.onResponse(RealWebSocket.kt:195)
at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:519)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
at java.lang.Thread.run(Thread.java:1012)
```

Reply:
That works!